qa/L0_simple_go_client/test.sh

```diff
@@ -55,26 +55,23 @@ git clone ${TRITON_REPO_ORGANIZATION}/client.git
 go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```

Comment:
I'm assuming this `@latest` is why this test is breaking or changing behavior every so often

Reply:
I think the reason was because we upgraded the go version itself. We could have searched for the `@latest` packages and fixed them to a version. But instead have modified the other test scripts like L0_grpc to work with newer behavior. 

go 1.22 has introduced various changes and I think it is ok to upgrade however, we will try fixing the version from future.